spec/factories/assignments.rb

```diff
@@ -21,6 +21,7 @@
     tokens_per_period 0
     unlimited_tokens false
     enable_test false
+    number_of_peer_reviews_per_group 3
```

Comment:
Should I make a constant like `Assignment::DEFAULT_PEER_REVIEW_COUNT` or something like that for this?